spec/policies/organisation_policy_spec.rb

```diff
@@ -19,15 +19,14 @@
       expect(subject).to permit(create(:user_in_organisation, role: 'admin'), build(:organisation))
     end
 
-    it "allows organisation admins to assign a user only to organisations within their organisation subtree" do
+    it "is forbidden for organisation admins even when assigning a user to organisations within their organisation subtree" do
```

Comment:
Still nitpicking on descriptions, but it's only because I'm hoping it's clear what this does to whomever looks at it for the first time without having much context. Perhaps saying:
`is forbidden for organisation admins to assign a user to organisations within their organisation subtree`
would be a bit clearer?